include/Game/Entities/DangoMushi.h

```diff
@@ -96,20 +96,21 @@ struct Obj : public EnemyBase {
 
 	// _00 		= VTBL
 	// _00-_2BC	= EnemyBase
-	u8 _2BC[0x4];                           // _2BC, unknown
+	FSM* mFsm;                              // _2BC
 	u8 _2C0;                                // _2C0
 	u8 _2C1;                                // _2C1
 	u8 _2C2;                                // _2C2
+	u8 _2C3;                                // _2C3
 	f32 _2C4;                               // _2C4, timer?
-	u8 _2C8[0x4];                           // _2C8, unknown
+	f32 mShadowScale;                       // _2C8, unknown
 	int _2CC;                               // _2CC
 	Vector3f _2D0;                          // _2D0
-	u8 _2DC[0xC];                           // _2DC, unknown
+	Vector3f mVel;                          // _2DC, unknown
```

Comment:
Use full variable naming please, mVelocity I assume

Reply:
Adding to this - ideally it's good to have variables like this be as descriptive as possible, and not overlap with existing stuff. EnemyBase already has an mVelocity, so this should be some other (specific) velocity, maybe rolling velocity, or claw velocity? If we don't really have enough info yet to decide, it's totally fine to leave it as `Vector3f _2DC` for now until we know better.